helm/extensions.bzl

```diff
@@ -116,3 +120,40 @@ helm = module_extension(
     implementation = _helm_impl,
     tag_classes = {"options": options},
 )
+
+def _helm_import_impl(module_ctx):
+    for mod in module_ctx.modules:
+        for repository in mod.tags.repository:
+            helm_import_repository(
+                name = repository.name,
+                chart_name = repository.chart_name,
+                repository = repository.repository,
+                sha256 = repository.sha256,
+                url = repository.url,
+                version = repository.version,
+            )
+
+helm_import = module_extension(
```

Comment:
Consider merging this with the module extension above by introducing the `repository` tag_class there.

This way, from a user perspective, this is could then be used like
```
helm = use_extension("@rules_helm//helm:extensions.bzl", "helm")
helm.options(...)
helm.repository(...)
helm.repository(...)
```